### PR TITLE
[FIX] pos_online_payment: translatable warning message

### PR DESCRIPTION
**Pull request by niyasraphy**
before this commit, the added user error and validation error is not translatable to end user language.

after this commit, user error and validation error will be show in users language

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/pos_online_payment/models/pos_payment_method.py [hunk 1]
```diff
@@ -114,5 +114,5 @@ def _get_or_create_online_payment_method(self, company_id, pos_config_id):
                     'company_id': company_id,
                 })
                 if not payment_method_id:
-                    raise ValidationError('Could not create an online payment method (company_id=%d, pos_config_id=%d)' % (company_id, pos_config_id))
+                    raise ValidationError(_('Could not create an online payment method (company_id=%d, pos_config_id=%d)' % (company_id, pos_config_id)))
```
Comment (caburj) [inline addons/pos_online_payment/models/pos_payment_method.py:117]:
```suggestion
                    raise ValidationError(_('Could not create an online payment method (company_id=%d, pos_config_id=%d)', company_id, pos_config_id))
```

Comment (niyasraphy) [reply addons/pos_online_payment/models/pos_payment_method.py]: thanks @caburj ,
updated the pr

File: addons/pos_online_payment/models/pos_payment_method.py [hunk 2]
```diff
@@ -114,5 +114,5 @@ def _get_or_create_online_payment_method(self, company_id, pos_config_id):
                     'company_id': company_id,
                 })
                 if not payment_method_id:
-                    raise ValidationError('Could not create an online payment method (company_id=%d, pos_config_id=%d)' % (company_id, pos_config_id))
+                    raise ValidationError(_('Could not create an online payment method (company_id=%d, pos_config_id=%d)' % company_id, pos_config_id))
```
Comment (caburj) [inline addons/pos_online_payment/models/pos_payment_method.py:117]: you'll need to get rid of the "%" operator and replace it with comma ",". From how I understand it, the translation function has a built in string interpolation.

Comment (niyasraphy) [reply addons/pos_online_payment/models/pos_payment_method.py]: updated the pr

**Review by caburj — APPROVED** [review]
robodoo r+

**Comment by caburj** [comment]
@niyasraphy Thank you for your contributions. I don't want to impose, but if it's not too much to ask, can you prefix your future dev branches with the odoo version that it targets? For example, in this PR, it would be named: `master-trnsltn_online_payment`

**Comment by niyasraphy** [comment]
@caburj sure, i will do from next pr ,
thanks for taking into right direction

**Pull request closed without merging**